components/server/src/orgs/organization-service.ts

```diff
@@ -645,11 +625,12 @@ export class OrganizationService {
                 settings.roleRestrictions = partialUpdate.roleRestrictions;
             }
 
-            if (partialUpdate.onboardingSettings?.welcomeMessage && settings.onboardingSettings?.welcomeMessage) {
-                if (!partialUpdate.onboardingSettings.welcomeMessage.featuredMemberId) {
-                    settings.onboardingSettings.welcomeMessage.featuredMemberId = undefined;
-                    settings.onboardingSettings.welcomeMessage.featuredMemberResolvedAvatarUrl = undefined;
-                }
+            if (
```

Comment:
This block should move out of the `mergeSettings`, and merged with the other welcomeMessage validation.

Reply:
Addressed in [`d9e1cf2` (#20626)](https://github.com/gitpod-io/gitpod/pull/20626/commits/d9e1cf2d375af659a24324f6e360bcd44f1affd9)